src/main/java/uk/gov/hmcts/divorce/divorcecase/model/SolicitorService.java

```diff
@@ -102,4 +102,13 @@
     )
     private String serviceSotFirm;
 
+    @CCD(
+        label = " ",
+        access = {DefaultAccess.class}
+    )
+    private String truthStatement;
+
+    public String getTruthStatement() {
+        return "I believe that the facts stated in the application are true.";
```

Comment:
`.readonly(SolicitorService::getTruthStatement)` along with above should work ?

Reply:
My current implementation seems to trip up one the SolicitorSubmitAoS Integration test because the truthStatement field gets populated by default in the test and it puts the whole SolicitorService instance in the returned data - which is different from the expected json return string

Reply:
I have fixed up 2 of the integration tests to match the solServiceTruthStatement which now gets returned in the response body.

Reply:
Also needed to fix 4 Functional tests that were impacted by the same issue